### PR TITLE
Test PR

### PR DESCRIPTION
**Pull request by BerthaBrenes**
This pull request is a test

### CODE REVIEW

**Review by estalvgs1999 — CHANGES_REQUESTED** [review]
I need a comment with current date ... ASAP :3

File: frontend/src/index.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ ReactDOM.render(
   document.getElementById('root')
 );
 //test error
+// comment PR
+// 18/08/2021 22:16
```
Comment (estalvgs1999) [inline frontend/src/index.js]: It's fine ... but we need it in yyyy/mm/dd format thx!

**Comment by estalvgs1999** [comment]
I can't approve your PR ... sorry 😢

File: frontend/src/index.js (hunk 1 above)
Comment (BerthaBrenes) [reply frontend/src/index.js]: Yes sure

**Comment by BerthaBrenes** [comment]
please re-open the PR, I already made the change you ask for

**Review by estalvgs1999 — APPROVED** [review]
Good job Bertha!! 
You've earned a double slice at Pizza Friday. 🍕

**Pull request merged**